src/Popover/Popover.js

```diff
@@ -78,14 +78,22 @@ export const styles = {
 };
 
 class Popover extends React.Component {
+  componentDidMount() {
+    if (this.props.updateReference) {
+      this.props.updateReference(() => {
```

Comment:
This block looks exactly like the one we have to handle the resize event.
Can we factorize the code?